RoosterPlanner.Api/Models/AvailabilityViewModel.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using RoosterPlanner.Models;
+using RoosterPlanner.Models.Types;
+
+namespace RoosterPlanner.Api.Models
+{
+    public class AvailabilityViewModel : EntityViewModel
+    {
+        public Guid? ParticipationId { get; set; }
+        public ParticipationViewModel Participation { get; set; }
+        public Guid ShiftId { get; set; }
+        public ShiftViewModel Shift { get; set; }
+        public AvailibilityType Type { get; set; }
+        public bool Preference { get; set; }
+
+        public static AvailabilityViewModel CreateVm(Availability availability)
+        {
+            if (availability == null)
+                return null;
+
+            AvailabilityViewModel vm = new AvailabilityViewModel
+            {
+                Id = availability.Id,
+                ParticipationId = availability.ParticipationId,
+                ShiftId = availability.ShiftId,
+                Type = availability.Type,
+                Preference = availability.Preference,
+                LastEditDate = availability.LastEditDate,
```

Comment:
Zijn deze twee velden nodig in de front-end?

Reply:
LastEditBy en RowVersion? Deze zijn toegevoegd (aan alle viewmodels) na ons gesprek zodat entiteiten (in de toekomst) eenvoudig geupdate kunnen worden.